nvflare/lighter/project.yml

```diff
@@ -106,6 +103,20 @@ builders:
             args:
               gpu_resource_key: gpu
 
+  - path: nvflare.lighter.impl.auth_policy.AuthPolicyBuilder
+    args:
+      orgs:
+        nvidia:
+          - relaxed
+      roles:
+        super: super user of system
+      groups:
+        relaxed:
+          desc: org group with relaxed policies
+          rules:
+            allow_byoc: true
+            allow_custom_datalist: true
+      disabled: false
```

Comment:
I recall we need to remove authorization.json.  Not sure if we will keep this for release.

Reply:
let's table this and discuss

Reply:
@yanchengnv @yhwen do you have any inputs on this? thanks

Reply:
@nvidianz is working on the re-engineering of admin authorization. Until that work is ready, we still need this to work for secure mode properly.